cli/config/compose/services/vsphere/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.3'
 services:
   vsphere:
-    image: "nimmis/vcsim:${vsphereTag}"
+    image: "nimmis/vcsim:${VSPHERE_VERSION}"
```

Comment:
The variables coming from Beats repo are following this name convention, although this vsphere is not including a supported-versions.yml file. I.e. apache:
https://github.com/elastic/beats/blob/master/metricbeat/module/apache/_meta/supported-versions.yml#L1-L3